packages/terra-overlay/src/Overlay.module.scss

```diff
@@ -11,7 +11,31 @@
     justify-content: center;
     outline: none;
     overflow: hidden;
-    z-index: 100;
```

Comment:
Should we keep this `z-index: 100` as a default? It seems this may be a breaking change if we remove the default z-index.

Reply:
I still have it as the default: https://github.com/cerner/terra-core/pull/1955/files#diff-5c1dfe9c477f29f0ae7f7738c5cb13e9R62

Reply:
ah ok, perfect!